modules/barcode/samples/barcode.cpp

```diff
@@ -0,0 +1,302 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+// Copyright (c) 2020-2021 darkliang wangberlinT Certseeds
```

Comment:
OpenCV **samples** are widely used in documentation, so these files don't have license headers

> except for the samples

https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure